### ISSUE TITLE
not able to generate the QR code

### ISSUE THREAD

**Issue by rpanjwani** [azure-ad-b2c/samples#161]
I'm hosting the TOTP sample and connected it to my Azure B2C setup by troubleshooting and following the minimal documentation here. I can't get the QR code to show up when the page is rendered by AD B2C on the user signup_signin journey. Not sure if this is a technical issue, or documentation needs to be vastly improved so it can make sense.

**Comment by JasSuri** [comment]
Is javascript enabled in the relying party file?

**Comment by rpanjwani** [comment]
yep, i've enabled it and my policies are set up pretty close to what's in the repo. Is there anything additional that needs to be done?

**Comment by JasSuri** [comment]
Are you referencing this html template in the content definition?
https://github.com/azure-ad-b2c/samples/blob/master/policies/custom-mfa-totp/source-code/AADB2C.MFA.TOTP/wwwroot/selfasserted-appfactor-registration.html

**Comment by rpanjwani** [comment]
yep, it's hosted here along with the service as is: https://samples318f.azurewebsites.net/selfasserted-appfactor-registration.html

**Comment by rpanjwani** [comment]
i've followed all the instructions including setting the two custom attributes
